lib/saml_idp/controller.rb

```diff
@@ -65,6 +65,7 @@ def encode_authn_response(principal, opts = {})
       session_expiry = opts[:session_expiry]
       encryption_opts = opts[:encryption] || nil
       signed_message_opts = opts[:signed_message] || false
+      signed_assertion_opts = opts[:signed_assertion] || true
```

Comment:
And it would be nice that if you have test for your option (spec/lib/saml_idp/controller_spec.rb).